panel/models/vtk/vtk_layout.ts

```diff
@@ -1,14 +1,136 @@
-import {PanelHTMLBoxView, set_size} from "../layout"
+import * as p from "@bokehjs/core/properties"
+
 import {div} from "@bokehjs/core/dom"
+import {clone} from "@bokehjs/core/util/object"
+import {HTMLBox} from "@bokehjs/models/layouts/html_box"
+
+import {PanelHTMLBoxView, set_size} from "../layout"
 
-import  {vtkns} from "./vtk_utils"
+import  {vtkns, VolumeType, majorAxis} from "./vtk_utils"
 
-export class VTKHTMLBoxView extends PanelHTMLBoxView{
+export abstract class AbstractVTKView extends PanelHTMLBoxView{
+  model: AbstractVTKPlot
   protected _vtk_container: HTMLDivElement
   protected _vtk_renwin: any
+  protected _orientationWidget: any
+  protected _widgetManager: any
+  protected _setting_camera: boolean = false
+
+  connect_signals(): void {
+    super.connect_signals()
+    this.connect(this.model.properties.data.change, () => {
+      this.invalidate_render()
+    })
+    this.connect(this.model.properties.orientation_widget.change, () => {
+      this._orientation_widget_visbility(this.model.orientation_widget)
```

Comment:
visibility is misspelled, but at least consistently so it won't hold up merge.

Reply:
Indeed I pretty good to misspelled it